domain-model-assistant/Assets/Components/Scripts/Diagram.cs

```diff
@@ -220,61 +221,15 @@ public void LoadJson(string cdmJson)
         _namesUpToDate = false;
     }
 
-    public void AddAttributes(GameObject sect, int i)
+    public void AddAttributesToSection(GameObject section)
```

Comment:
Does this method belong here? Why isn't this in `CompartmentedRectangle`?

Reply:
This method is being used in the `CreateSection()` method of `CompartmentedRectangle` class

Reply:
This also needs to be discussed in the future :rocket: